examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs

```diff
@@ -152,10 +151,10 @@ public static void Main(string[] args)
             "Performance Max retail campaign.";
 
         /// <summary>
-        /// A factory that creates MutateOperation's for removing an existing tree of
-        /// AssetGroupListingGroupFilter's.
+        /// A factory that creates MutateOperations for removing an existing tree of
+        /// AssetGroupListingGroupFilters.
         ///
-        /// AssetGroupListingGroupFilter's must be removed in a specific order: all of the children
+        /// AssetGroupListingGroupFilters must be removed in a specific order: all of the children
         /// of a filter must be removed before the filter itself, otherwise the API will return an
         /// error.
         ///
```

Comment:
Is the [resources](https://github.com/googleads/google-ads-dotnet/blob/6f0678260aa7913865f1c0aab24bd61ccaef1d06/examples/ShoppingAds/AddPerformanceMaxProductListingGroupTree.cs#L166) instance variable actually used? Can this be removed?

Reply:
+1 it doesn't appear to be used anywhere.